cap-tempfile/src/lib.rs

```diff
@@ -101,6 +101,15 @@ impl TempDir {
         Err(Self::already_exists())
     }
 
+    /// Make this directory persistent.
+    ///
+    /// This corresponds to [`tempfile::TempDir::into_path`].
+    ///
+    /// [`tempfile::TempDir::into_path`]: https://docs.rs/tempfile/latest/tempfile/struct.TempDir.html#method.into_path
+    pub fn persist(mut self) -> io::Result<Dir> {
+        Ok(self.dir.take().unwrap())
+    }
```

Comment:
Would it make sense to call this function `into_dir`, for closer analogy with the tempfile API?

Reply:
Sure, done!